about.html

```diff
@@ -65,6 +65,54 @@
                         <div id="releaseContent" class="panel-collapse collapse">
                             <div class="panel-body">
                                 <div class="panel-group">
+
+                                    <div class="panel panel-default"></div>
+                                        <div class="panel-heading">
+                                            <p class="panel-title aboutHeaders">
+                                                <a href="#v7_1_0Content" data-toggle="collapse">
+                                                    GRNsight v7.1.0
+                                                </a>
+                                            </p>
+                                        </div>
+                                        
+                                        <div id="v7_1_0Content" class="panel-collapse collapse">
+                                            <div class="panel-body">
+                                                GRNsight v7_1_0 includes the following new features and bug fixes:
```

Comment:
I think we need dots here rather than underscores… (plus the version number)

```suggestion
                                                GRNsight v7.1.1 includes the following new features and bug fixes:
```

Reply:
Done